From: Roki Buildroot <build@roki.local>
Subject: [PATCH] build: omit UCM utilities when UCM is disabled

The robot image does not retain ALSA card state and does not use UCM. Buildroot
does not install alsactl or alsaucm when they are disabled, but upstream still
compiles them unconditionally. Omit the unused subdirectories so they cannot
link against UCM symbols intentionally absent from alsa-lib.

---
 Makefile.am | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index f2860032..289b4872 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,6 +1,6 @@
 AM_CPPFLAGS=-I$(top_srcdir)/include
 
-SUBDIRS = include alsactl utils m4 po alsa-info
+SUBDIRS = include utils m4 po alsa-info
 if ALSAMIXER
 SUBDIRS += alsamixer
 endif
@@ -26,8 +26,5 @@ if HAVE_SEQ
 SUBDIRS += seq
 endif
-if HAVE_UCM
-SUBDIRS += alsaucm
-endif
 if HAVE_TOPOLOGY
 SUBDIRS += topology
 endif
-- 
2.51.0
